sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -106,10 +97,6 @@ public DefaultAndroidEventProcessor(
 
   private @NotNull Map<String, Object> loadContextData() {
     Map<String, Object> map = new HashMap<>();
-    String[] proguardUUIDs = getProguardUUIDs();
-    if (proguardUUIDs != null) {
-      map.put(PROGUARD_UUID, proguardUUIDs);
```

Comment:
var `PROGUARD_UUID` can be removed from this class now

Reply:
Thanks, missed that.